src/Redux/Actions/verify.action.js

```diff
@@ -0,0 +1,27 @@
+import constants from '../constants';
+import apis from '../../Api';
+
+const VerifyUser = token => {
+	return async dispatch => {
+		dispatch({
+			type: constants.VERIFY_PENDING,
+			pending: true,
+		});
+		try {
+			const result = await apis.verifyUser(token);
+			if (result.status === 200) {
+				return dispatch({
```

Comment:
@lemoisoN which status can come here apart from 200 if error then it will be caught by  catch so I think we don't need this statement